packages/sirius-web/backend/sirius-web-sample-application/src/test/java/org/eclipse/sirius/web/sample/tests/integration/task/GanttControllerIntegrationTests.java

```diff
@@ -67,4 +67,16 @@ public void givenTaskModelThenGanttRepresentationCreationSucceeds() {
         String representationId = JsonPath.read(result, "$.data.createRepresentation.representation.id");
         assertThat(representationId).isNotNull();
     }
+
+
+    @Test
+    @DisplayName("Given a task model, when a create representation is performed, then the representation is created")
+    @Sql(scripts = { "/scripts/initialize.sql" }, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+    @Sql(scripts = { "/scripts/cleanup.sql" }, executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD, config = @SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED))
+    public void givenGanttThenTasksMutationsSucceed() {
+
+        this.givenTaskModelThenGanttRepresentationCreationSucceeds();
```

Comment:
Why do you call the other test in this one...

Reply:
It is a mistake. I started to write a test, but I prefer wait for the new API to be able to easily add a semantic model programmatically in my test in addition to programmatically add a view model.
I will remove this test waiting for the new API.